core/Cargo.toml

```diff
@@ -86,11 +86,13 @@ strsim = "0.10.0"
 pretty_assertions.workspace = true
 assert_matches.workspace = true
 criterion.workspace = true
-pprof = { workspace = true, features = ["criterion", "flamegraph"] }
 nickel-lang-utils.workspace = true
 similar.workspace = true
 test-generator.workspace = true
 
+[lints.clippy]
+doc_lazy_continuation = { level = "allow" }
```

Comment:
Why turn this off? I don't see it firing, but even if it does it seems like a useful lint

Reply:
I agree that it is a useful hint. There are 49 places where the warning occurs (with clippy 0.1.80), so I assumed that the documentation violating the lint is a deliberate style choice. I can drop that commit.